src/canvas/App.tsx

```diff
@@ -0,0 +1,18 @@
+import { Renderer } from './Renderer';
+import { Document } from './Document';
+import { PathSelection } from './PathSelection';
+
+class App {
```

Comment:
so we now have 2 Apps? the react one and the canvas one?

Reply:
Yeah, I will try to think of another name for for the new class.

Reply:
maybe this one should be AppModels?

Reply:
I changed the name of this class to AppController

Reply:
ok